spec/views/verify/review/new.html.slim_spec.rb

```diff
@@ -7,7 +7,7 @@
     before do
       user = build_stubbed(:user, :signed_up)
       allow(view).to receive(:current_user).and_return(user)
-      allow(view).to receive(:idv_params).and_return(
+      @idv_params = {
```

Comment:
oooh maybe it is? I am not sure. Looks like we are doing both right now. which do you prefer?